integrations/google/server.go

```diff
@@ -24,35 +24,35 @@ const (
 	gmailWebhookPath = "/gmail/notif"
 )
 
-func Start(l *zap.Logger, mux *http.ServeMux, v sdkservices.Vars, o sdkservices.OAuth, d sdkservices.Dispatcher) {
+func Start(l *zap.Logger, muxNoAuth *http.ServeMux, muxAuth *http.ServeMux, v sdkservices.Vars, o sdkservices.OAuth, d sdkservices.Dispatcher) {
```

Comment:
Please add a function documentation comment for `Start` to explain that it's currently the only integration that uses `muxAuth`, what `muxAuth` is (in comparison to the usual `muxNoAuth`), and why it's needed at all. In other words, the context and description of this PR as a comment here, to help future maintainers of this code.